Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.GPU.sb.in

```diff
@@ -790,7 +791,7 @@
         SYS_chdir
         SYS_close
         SYS_close_nocancel
-        SYS_connect
+        SYS_connect        
```

Comment:
Unintentional change?

Reply:
Wow -- don't even know how that happened. Let me fix it.